docs/instructions.md

```diff
@@ -29,32 +47,46 @@ ___
 
 ### Part One: Implement Backend API
 
-Implement an API sever that provides the following service over gRPC:
-[proto-file](../api/v1/logistics.proto).
+Implement an API sever that provides the following service.
+
+- If you know gRPC: [proto-file](../api/v1/logistics.proto)
+- Alternative HTTP: [swagger-file](../api/v1/logistics.swagger.json)
 
-- The solution should output a log message to STDOUT
-once per second with the number of received messages per second.
+> ! The solution must continuously output a log message to STDOUT every second.
+> Each log message should include the total number of messages received during
+> that second.
```

Comment:
Using Markdown quotes combined with how GitHub renders them does not bring the attention you intended to this information. I would use a normal paragraph.

```suggestion
The solution must continuously output a log message to STDOUT every second.
Each log message should include the total number of messages received during
that second.
```

Reply:
Made it even better then - **Bold**